content/events-reference/events-reference-serenity/resources/product-updated.yml

```diff
@@ -0,0 +1,132 @@
+type: object
+properties:
+  events:
+    type: array
+    items:
+      type: object
+      properties:
+        action:
+          type: string
+          description: Action that triggered the event
+        event_id:
+          type: string
+          description: Event identifier
+        event_date:
+          type: string
+          description: Event datetime
+        author:
+          type: string
+          description: Author of the event
+        author_type:
+          type: string
+          description: Type of the author "ui" or "api"
+        pim_source:
+          type: string
+          description: Pim source
+        data:
+          type: object
+          properties:
+            identifier:
+              type: string
+              description: Product identifier, i.e. the value of the only `pim_catalog_identifier` attribute
+            enabled:
+              type: boolean
+              description: Whether the product is enable
+            family:
+              type: string
+              description: Family code from which the product inherits its attributes and attributes requirements
+            categories:
+              type: array
+              description: Codes of the categories in which the product is classified
+              items:
+                type: string
+            groups:
+              type: array
+              description: Codes of the groups to which the product belong
+              items:
+                type: string
+            parent:
+              type: string
+              description: Code of the parent product model when the product is a variant (only available since the 2.0). This parent can be modified since the 2.3.
+            values:
+              type: object
+              properties:
+                attributeCode:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      scope:
+                        type: string
+                        description: Channel code of the product value
+                      locale:
+                        type: string
+                        description: Locale code of the product value
+                      data:
+                        type: object
+                        description: Product value
+                      linked_data:
```

Comment:
We have the linked_data by default in the event payload?

Reply:
No. I will remove it from the event api doc